extensions/wikia/AdEngine/js/slot/floatingMedrec.js

```diff
@@ -0,0 +1,88 @@
+/*global define*/
+define('ext.wikia.adEngine.slot.floatingMedrec', [
```

Comment:
Looks much lighter. Sometimes I have the feeling too light :japanese_ogre: The notes below can be used as QA notes or/and as something to check before moving to testing phase. 

I can't find RTL support in the new implementation. Does it work out-of-the-box and doesn't need code implementation?

~~I can see in the new code calculations connected to global navigation but in the old code it counted only when global nav was fixed. How does it change behavior of the ad on high screens when the global nav isn't fixed?~~ Oh, it's always fixed as far as I can quickly check now... :smile_cat: